docker/k8s/enclave-manager-deployment.yaml

```diff
@@ -14,8 +14,9 @@ spec:
         app: tee
     spec:
       containers:
-      - image: sammyne/hyperledger-tcf
+      - image: tcf-dev:latest
         name: tee
+        imagePullPolicy: Never
```

Comment:
Do we need to set image pull policy to Never always or is it a temporary approach for now?

Reply:
It is needed now so that Kubernetes doesn't attempt to pull the image since we are using local built.  Later on when we plan to host the Avalon docker image in a external repository, imagePullPolicy need not be set to never

Reply:
Other way to handle this would be to change the tag from `:latest` to either `:local` or `:<BUILD_VERSION>`. The `:<BUILD_VERSION>` is auto generated for every build, this is preferable. Kubernetes tries to pull the images if the tag is `:latest`.